sass/CategoryFacet/_CategoryFacetValues.scss

```diff
@@ -0,0 +1,94 @@
+@import '../Variables';
+
+$active-text-opacity: 0.6;
+
+// TODO: remove when old values are removed
+.coveo-category-facet-values,
+.coveo-category-facet-search-container,
+.coveo-category-facet-more-less-container {
+  display: none;
+}
+
+.coveo-dynamic-category-facet-values {
+  list-style: none;
+  padding: 10px 0;
+  margin: 0;
+
+  &.coveo-with-space .coveo-dynamic-category-facet-value  {
+    padding-left: 20px;
+
+    &.coveo-with-space {
+      padding-left: 45px;
+    }
+  }
+}
+
+.coveo-dynamic-category-facet-collapsed {
+  .coveo-dynamic-category-facet-value:not(.coveo-selected) {
+    display: none;
+  }
+}
+
+.coveo-dynamic-category-facet-value,
```

Comment:
Is there some repetition between these (and, actually this whole file), and the "standard" DynamicFacet values ?

Could we extract some mixins between the two types of facet values that we could leverage in both files ?

Reply:
A few things but yes, done